lhctl/internal/external_event.go

```diff
@@ -73,13 +80,17 @@ Returns a list of ObjectId's that can be passed into 'lhctl get externalEvent'.
 	returned. The time bound applies to the time that the ExternalEvents
 	were created.
 `,
+	Args: cobra.MaximumNArgs(1),
```

Comment:
it seems that we can use`cobra.ExactArgs(1)` instead, right?  then we can remove the validation on line [L#87](https://github.com/littlehorse-enterprises/littlehorse/pull/1219/files#diff-ef8ee86529c6a3aa1c4971184d98231012215689c15324dfbc5d79f3705f9d9fR86)

Reply:
Oh! nvm.. `externalEventDef` is optional